mapiproxy/servers/default/nspi/dcesrv_exchange_nsp.h

```diff
@@ -95,6 +95,7 @@ struct EphemeralEntryID {
 
 #define	EMSABP_DN	"/guid=%08X%04X%04X%02X%02X%02X%02X%02X%02X%02X%02X"
 #define	EMSABP_ADDRTYPE	"EX"
+//#define	EMSABP_ADDRTYPE	"SMTP"
```

Comment:
Could you remove this line? I don't see the point of adding this commented code.